src/content/overview/observability/observability-platform-api/_index.md

```diff
@@ -0,0 +1,105 @@
+
+Upon reaching the correct ingress, the request will need to be authenticated before being forwarded. This is done through a nginx annotation : `nginx.ingress.kubernetes.io/auth-url` which value is the customer OIDC provider ingress' url. As the requests coming from Grafana forwards the user's OAuth identity through the use of a token, the Giant Swarm ingress only make sure that the incoming request's token matches an allowed user in the OIDC provider.
+
+Moreover, the ingress will verify that the incoming requests are using the `X-Scope-OrgId` HTTP headers (tenant information). If not, the request will be denied.
+
+Upon authentication, the request is then forwarded to either the `loki-gateway` or `mimir-gateway` (depending if the query is looking for logs or metrics) components which are nginx servers respectively fronting the loki and mimir clusters. Those gateway components then finally forward the request to the read components.
+
+That's it, the user now has access to the query's output.
+
+#### Write path
+
+Let's consider the following situation : a customer's logging agent running on his premises want to send its data to the Giant Swarm Observability Platform.
+
+The workflow is quite similar as the one described for the read path. The main difference is in the fact that the requests will go through another ingress in the monitoring namespace which fronts a specifically tuned alloy instance, deployed via the [alloy-gateway-app](https://github.com/giantswarm/alloy-gateway-app).
+
+Once authenticated, the request will thus go through the `alloy-gateway` which will forward it to the `loki-gateway` and then in turn the `loki-write` components.
+
+This additional step was added to ensure that we support OTLP.
+
+## Adding datasources
+
+In order for one to use the observability-platform API's read path, the most straight-forward way is to add a datasource in one's Grafana pointing towards it. Here is a step-by-step guide on how to do it :
+
+* in the `Connection` section, write the observability-platform API url which always be `https://observability.<domain_name>`. Replace the `<domain_name>` placeholder with the actual domain name of your installation.
+
+![datasource url](./datasource-url.png)
+
+Please note that if you're adding a Mimir or Prometheus datasource, you will have to add the `/prometheus` suffix to the url (i.e the final url will be of the following form : `https://observability.<domain_name>/prometheus`).
+
+* In the `Authentication` section, select the `Forward OAuth Identity` option.
+
+![datasource authentication](./datasource-authentication.png)
+
+* Sill in the `Authentication` section, add a HTTP header :
```

Comment:
```suggestion
3. Fill in the `Authentication` section, add the required `X-Scope-OrgID`. Make sure the value of the `X-Scope-OrgID` header is an existing tenant that actually holds the data you're interested in. For example, to access Giant Swarm managed logs the value of this  header should be `giantswarm`, for metrics it should be `anynomous`. You can learn more about our tenant concept in [multi-tenancy in the observability platform.]({{< relref "/tutorials/observability/multi-tenancy/" >}})
```

Reply:
What is the value of the tenant header we need for the default org? That might for both be the way better example than exposing the giant swarm one :D Also we might want to think about how we can secure individual tenant reads, so that a team that shouldn't have access to our giantswarm data suddenly can see them through setting up their own grafana and using the api only because another team had the api for an actual valid reason.